Exercise_B/index.html

```diff
@@ -17,18 +18,92 @@
     <link rel="stylesheet" href="./assets/style/stylesheet.css" />
   </head>
 
+
+
   <body>
     <!--Header (flower shop logo block element)-->
+    <header> <img src="./assets/images/logo.png" alt="logo"></header>
+
+
     <!--Banner Jumbotron of the image-->
+    <div class="intro">
+      <h1> Beautiful Flowers </h1>
+
+      <button> Learn more </button>
+      <div class="background">
+
+      </div>
+    </div>
+
+
 
     <!--Four containers with img / text-->
+
     <!-- 4 containers -->

```

Comment:
Hello Chloe 
For this you would need to change your Grid 
You can refer to this: [https://getbootstrap.com/docs/5.0/layout/grid/](url)